build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.3'
+        classpath 'com.android.tools.build:gradle:2.2.0'
```

Comment:
nice one, this update doesn't make sense in this pull request though. I prefer having that in a separate one.